sdk/attestation/attestation/src/utils/blobConvert.browser.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
let's not have the blob stuff in this PR.

Reply:
gone.